### PR TITLE
"View Match" button

### LINKED ISSUE #146: "View Match" button doesn't work yet
In `StatusCardMatched.vue` , the "View Matched" button doesn't work yet.

It's unclear from the Figma what should happen when "View Matched" is clicked. Perhaps we could link to the unmatching modal below and rename the button to "Unmatch".

(Note that we can already view the matched teacher's name in the status card.)

image

### PR DESCRIPTION
**Pull request by xuanjane**
# "View Match" button on individual student profile page

Issue #146

## Description

On individual student profile page, changed View Match button to Unmatch button.
Implemented with full functionality.

## Test Steps

1. Match a student and navigate to his/her individual student profile page
2. Use the "unmatch" button to unmatch/drop out the student
3. Verify that the student is unmatched/dropped out successfully. Corresponding teacher is unmatched as well. Unmatch/Drop out reason is reflected in the student history.

### CODE REVIEW

File: frontend/src/components/modals/ModalUnmatch.vue [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    },
+    filteredReasonDataArray() {
+      return this.API_droppedReason.filter((option) => {
+        return option.Reason.toLowerCase().includes(
+          this.selected.Reason.toString().toLowerCase()
+        );
+      });
+    },
+  },
+  async mounted() {
```
Comment (aelbore) [inline frontend/src/components/modals/ModalUnmatch.vue]: you can remove `async` here

**Review by aelbore — APPROVED** [review]
looks good to me

**Review by nondescryptid — APPROVED** [review]
Tested, behaviour was as expected - student is unmatched along with the teacher

File: frontend/src/components/modals/ModalUnmatch.vue (hunk 1 above)
Comment (xuanjane) [reply frontend/src/components/modals/ModalUnmatch.vue]: okay sure, thank you!

**Pull request merged**